cmd/operator/main.go

```diff
@@ -46,7 +46,7 @@ func (t *tags) Set(value string) error {
 	for _, pair := range pairs {
 		splitPair := strings.Split(pair, "=")
 		if len(splitPair) != 2 {
-			return fmt.Errorf("Pair %v is malformed. Key value pairs must be in the form of \"key=value\". Multiple pairs must be comma separated.")
+			return fmt.Errorf("pair %q is malformed; key-value pairs must be in the form of \"key=value\"; pultiple pairs must be comma-separated", value)
```

Comment:
tests complained about a missing second argument to serve as the formatted value